index.bs

```diff
@@ -377,12 +377,7 @@ The Web Authentication API is defined by the union of the Web IDL fragments pres
 <xmp class="idl">
     [SecureContext]
     interface WebAuthentication {
-        Promise<ScopedCredentialInfo> makeCredential(
-            RelyingPartyUserInfo                 accountInformation,
-            sequence<ScopedCredentialParameters> cryptoParameters,
-            BufferSource                         attestationChallenge,
-            optional ScopedCredentialOptions     options
-        );
+        Promise<ScopedCredentialInfo> makeCredential(MakeCredentialOptions options);
```

Comment:
But, I do not like the term `request` here because that dict instance is not a "request" in an of itself.  I much prefer just using `options` or `parameters` here (see https://github.com/w3c/webauthn/pull/398#issuecomment-294038181).  If I were to implement, I'd find `request` misleading/confusing.